.circleci/config.yml

```diff
@@ -4,21 +4,68 @@ aliases:
     docker:
       - image: circleci/python:3.6.5-stretch-node-browsers
     working_directory: ~/fun
+  # Activate dind
+  - &dind
+    setup_remote_docker:
+        docker_layer_caching: true
   - &ci_env
     run:
       name: Define Environment Variable at Runtime
       command: |
         echo 'export IMAGE="arnold:$(tr -d '\n' < VERSION)"' >> $BASH_ENV
         source $BASH_ENV
-  # Activate dind
-  - &dind
-    setup_remote_docker:
-        docker_layer_caching: true
+  - &docker_load
+    run:
+      name: Load docker image
+      command: |
+        docker load < /tmp/docker/images/arnold.tar
+        docker images
+  # Restore workspace
+  - &attach_workspace
+    attach_workspace:
+      # Must be absolute path or relative path from working_directory
+      at: /tmp/docker
 
 ci_env:
 
 version: 2
 jobs:
+
+  ansible_lint:
+    <<: *defaults
+    steps:
+      - checkout
+      - *dind
+      - *ci_env
+      - *attach_workspace
+      - *docker_load
+      - run: |
+          docker run --rm -it \
+            -e OC_LOGIN=false \
+            "${IMAGE}" \
+            bin/lint ansible
+
+  bash_lint:
+    <<: *defaults
+    steps:
+      - checkout
+      - *dind
+      - *ci_env
+      - *attach_workspace
+      - *docker_load
+      - run: |
+          docker run --rm -it \
+            -e OC_LOGIN=false \
+            "${IMAGE}" \
+            bin/lint bash
+
+  docker_lint:
+    <<: *defaults
+    steps:
+      - checkout
+      - *dind
+      - run: docker run --rm -i hadolint/hadolint < Dockerfile
```

Comment:
@jmaupetit should we add this to fun-platform and richie?

Reply:
I think yes